package.json

```diff
@@ -8,6 +8,10 @@
   "license": "UNLICENSED",
   "main": "index.js",
   "typings": "index.d.ts",
+  "files": [
+      "/include/*.js",
```

Comment:
like `cli`, I think this can be `/include/` and `/src/`, though double check the output of `npm pack` to verify that.

Reply:
Fixed in c86230c.